src/pybind/mgr/orchestrator/module.py

```diff
@@ -2239,16 +2239,30 @@ def _tuned_profile_ls(self, format: Format = Format.plain) -> HandleCommandResul
             return HandleCommandResult(stdout=out)
 
     @_cli_write_command('orch tuned-profile add-setting')
-    def _tuned_profile_add_setting(self, profile_name: str, setting: str, value: str) -> HandleCommandResult:
-        completion = self.tuned_profile_add_setting(profile_name, setting, value)
-        res = raise_if_exception(completion)
-        return HandleCommandResult(stdout=res)
+    def _tuned_profile_add_setting(self, profile_name: str, settings: List[str]) -> HandleCommandResult:
+        if len(settings) % 2 != 0:
+            raise OrchestratorValidationError(
+                f"Invalid number of arguments. Each setting must have a corresponding value. "
+                f"Received: {settings}"
+            )
+        results = []
+        for i in range(0, len(settings), 2):
+            setting, value = settings[i], settings[i + 1]
+            completion = self.tuned_profile_add_setting(profile_name, setting, value)
+            res = raise_if_exception(completion)
+            results.append(str(res))
```

Comment:
I think this would work better if split on commas and require an `=` between the setting and value rather than just having every other string be a setting or value. There's an example of that here https://github.com/ceph/ceph/blob/c0451e1ca2b471dbdadb13cd64178e3dd83b4c62/src/pybind/mgr/orchestrator/module.py#L1370-L1374 that I think could be followed